book/src/products/mosaic-overview.md

```diff
@@ -0,0 +1,22 @@
+# Mosaic Overview
+
+Mosaic is Composable Finance’s cross-chain transfer availability layer that combines a [dynamic fee model](https://medium.com/composable-finance/the-dynamic-fee-model-powering-mosaics-transfer-availability-layer-f91011309592), [liquidity forecasting](https://medium.com/composable-finance/liquidity-forecasting-in-mosaic-part-iv-machine-learning-based-methods-17e8f2e5de14), [active liquidity management](https://composablefi.medium.com/understanding-mosaics-active-management-e1894fc90a00), and passive liquidity rebalancing to create a transfer availability layer. As a proactive bridging infrastructure, Mosaic leverages a network of existing bridges to manage its LP vaults on different layers ensuring the ability to transfer assets cross-chain regardless of volume or size.
+
+Mosaic launch is planned in three phases. Phase 1 is a Proof of Concept that only connects Ethereum, Arbitrum, and Polygon to gather data on transfer sizes and fees. Phase 2 delivers the core features of Mosaic, including dynamic fees, passive liquidity forecasting and rebalancing, and active liquidity management. Phase 3 decentralizes the relayer with the Threshold-Signature-Scheme and allows for on-chain dispute resolution. 
```

Comment:
Mosaic launch was planned in three phases. Phase 1 has been completed, which is the current version consisting of a Proof of Concept that only connects Ethereum, Arbitrum, and Polygon to gather data on transfer sizes and fees.